<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RK txt </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>611</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnRU">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnENG">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openRead_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть для чтения</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>621</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxLang">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Язык</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLight">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Светлая</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDark">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Темная</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxStyle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Стиль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/folderup.png</normaloff>:/img/folderup.png</iconset>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>140</y>
      <width>341</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/iconlist.png</normaloff>:/img/iconlist.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>90</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/list.png</normaloff>:/img/list.png</iconset>
    </property>
   </widget>
   <zorder>groupBoxStyle</zorder>
   <zorder>groupBoxLang</zorder>
   <zorder>groupBox</zorder>
   <zorder>textEdit</zorder>
   <zorder>open_pushButton</zorder>
   <zorder>save_pushButton</zorder>
   <zorder>new_pushButton</zorder>
   <zorder>close_pushButton</zorder>
   <zorder>help_pushButton</zorder>
   <zorder>radioBtnRU</zorder>
   <zorder>radioBtnENG</zorder>
   <zorder>openRead_pushButton</zorder>
   <zorder>radioButtonLight</zorder>
   <zorder>radioButtonDark</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>listView</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
